src/Aspire.Hosting.Python/PythonProjectResourceBuilderExtensions.cs

```diff
@@ -60,6 +60,8 @@ public static class PythonProjectResourceBuilderExtensions
     public static IResourceBuilder<PythonProjectResource> AddPythonProject(
         this IDistributedApplicationBuilder builder, string name, string projectDirectory, string scriptPath, params string[] scriptArgs)
     {
+        ArgumentNullException.ThrowIfNull(builder);
```

Comment:
Maybe add for `scriptArgs` also.

Reply:
> Maybe add for `scriptArgs` also.

We do this check on line 118.
And since we don’t use it directly in line 64, the check is not needed in this case.
But I can add a check for the others if you think it's necessary!